capstone/src/main/java/com/google/univiz/SuggestionData.java

```diff
@@ -0,0 +1,4 @@
+package com.google.univiz;
+
+/** SuggestionData represents the college data specific to the Search feature */
+public class SuggestionData {}
```

Comment:
Would we benefit from using `AutoValue` for this?

Reply:
yes :) However since it is just a placeholder here, I did not incorporate AutoValue in this PR